server/eventHandlers/storage.ts

```diff
@@ -19,6 +20,11 @@ export default class extends IEventHandler {
    */
   public async handle(event: CWEvent) {
     log.trace(`Received event: ${JSON.stringify(event, null, 2)}`);
+    if (this._excludedEvents.includes(event.data.kind)) {
+      log.trace('Skipping event!');
```

Comment:
Should we make the log statement say what kind of event was skipped?

Reply:
The trace is immediately after the "received event" so I assumed we'd know from that, but we could make a more explicit one if we wanted. Remember though that we're going to be skipping the vast majority of events (based on the statistics above), so we might not wanna worry about it at all, as printing them would be spammy.

Reply:
Agree that printing would probably be spammy...would make sense to hide altogether too

Reply:
It's a `trace` level log so it wont be shown unless you specifically change it to show trace.